### PR TITLE
test certs - cherry-pick PR325 on v2

### PR DESCRIPTION
**Pull request by dmitris**
PR copies the changes in #325 onto the v2 branch. See issue #324 for background.

### CODE REVIEW

**Review by johanbrandhorst — APPROVED** [review]

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Wait a sec, shouldn't this be deleting the existing certs too?

**Comment by dmitris** [comment]
> Wait a sec, shouldn't this be deleting the existing certs too?

done in 1b508da

File: grpctesting/interceptor_suite.go [hunk 1]
```diff
@@ -156,3 +162,45 @@ func (s *InterceptorTestSuite) TearDownSuite() {
...
+	notAfter := notBefore.Add(time.Hour)
+	serialNumberLimit := new(big.Int).Lsh(big.NewInt(1), 128)
+	serialNumber, err := rand.Int(rand.Reader, serialNumberLimit)
+	if err != nil {
+		return nil, nil, err
+	}
+	template := x509.Certificate{
+		SerialNumber: serialNumber,
+		Subject: pkix.Name{
+			CommonName: "Certify Test Cert",
```
Comment (johanbrandhorst) [inline grpctesting/interceptor_suite.go:183]: Ah, doh, we should probably change this too :sweat_smile:

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Pull request merged**